src/cpuid/src/template/intel/mod.rs

```diff
@@ -50,5 +55,48 @@ pub(crate) fn msrs_to_save_by_cpuid(cpuid: &CpuId) -> HashSet<u32> {
         leaf_0x7::index0::ebx::MPX_BITINDEX,
         [MSR_IA32_BNDCFGS]
     );
+
+    // MTRR
+    cpuid_msr_dep!(
+        0x1,
+        0,
+        edx,
+        leaf_0x1::edx::MTRR_BITINDEX,
+        [
+            0x200, // IA32_MTRR_PHYSBASE0
```

Comment:
nit: you can use a range here as well.

Reply:
Not quite. The structure is not regular. 0x200-0x20f are contiguous, but then there are holes. We could encode 0x200-0x20f as a range and the rest as individual values, but then the macro needs to be more clever to understand it. Any suggestions?

Reply:
Ah, maybe you meant splitting into 2 macro calls, one with a range, one with separate MSRs?

Reply:
done

Reply:
yup, that's what I meant. Looks good!